lib/inspector.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+const assert = require('assert');
+const connect = process.binding('inspector').connect;
+const EventEmitter = require('events');
+
+assert(connect, 'Inspector is not available');
+
+exports.Session = class extends EventEmitter {
+  constructor() {
+    super();
+    this._connection = connect((message) => this._onMessage(message));
+    this._nextId = 1;
+    this._messageCallbacks = new Map();
+  }
+
+  _onMessage(message) {
+    const parsed = JSON.parse(message);
+    if (parsed['id']) {
```

Comment:
No. It was more because this is coming from JSON deserialization so the "object" is more of a "map"